commands/Fun/say.js

```diff
@@ -0,0 +1,19 @@
+const { Command } = require("klasa");
+
+class Say extends Command {
+  constructor(...args) {
+    super(...args, {
+      description: "Make the bot say something",
+      usage: "<message:string>",
+      permissionLevel: 4,
+      aliases: ["echo"]
+    });
+  }
+
+  async run(msg, [args]) {
+    msg.delete().catch(O_o=>{}); 
+    return msg.channel.send(args);
```

Comment:
Please use the disableEveryone option in send. That will prevent issues and @ everyone spam

Reply:
Done